lux/config/runtime.exs

```diff
@@ -16,6 +16,7 @@ if config_env() in [:dev, :test] do
   config :lux, :api_keys,
     alchemy: env!("ALCHEMY_API_KEY", :string!),
     openai: env!("OPENAI_API_KEY", :string!),
+    together: env!("TOGETHER_API_KEY"),
```

Comment:
For CI and tests to pass we need to set this via the .test.envrc file as well (even if with a fake value).
![image](https://github.com/user-attachments/assets/838c89fe-e15c-4c7f-a0a5-b3115e6c63ec)

Reply:
There is no .test.envrc file, you mean .dev.envrc ?